server/data/licenceClient.ts

```diff
@@ -119,8 +151,9 @@ module.exports = {
     }
 
     return db.query(query)
-  },
+  }
 
+  // eslint-disable-next-line class-methods-use-this
```

Comment:
would be tempted to move to inject `db` in the future, then we can use normal mocking in the db tests and not have these warnings to suppress. It's a bit random we actually require it directly